content/solutions/altinn-studio/functional/text/_index.md

```diff
@@ -41,3 +41,117 @@ Where the first param is the language key and the second is the language element
 - Keys are written in lowercase with underscores (example_key)
 
 {{<figure src="oversetting.png?width=1000" title="Editor for oversetting av tekster">}}
+
+# Language best practice
+
+## Summary
+
+- Name should describe the content of the value.
+- Section should describe the context (might be page, container or component).
+- If your name also describes the page, container or component, you *might* want to create a new section.
+
+## Keys (properties)
+
+Every *key*, or *property*, has a *name* and a *value*.
+
+- Names should be **short** and **not context based** (context should be in section)
+- Names should not describe presentational functionality like modal or popover.
+
+Example:
+
+```ini
+[good_example_1]
+ready_to_deploy_title_false = Text...
+[bad_example_1]
+service_is_ready_to_deploy_title_false = Text...
+
+[good_example_2]
+repo_changes_is_invisible = Text...
+[bad_example_2]
+changes_made_by_others_in_your_organisation_is_not_visible_in_altinn_studio = Text...
+```
+
+### Text parts
+
+If your text has several parts, suffix with "part1", "part2", "partN".
+
+> It is currently not possible to use variables or HTML code in text.
+
+Example:
+
+```ini
+[deploy_to_test]
+error_there_is_something_wrong_with_your_environment_part1 = Det er noe galt med ditt
```

Comment:
Long example key names not following the rule saying we should have short keys.
Should be "error_environment_failure_part1" or something like that.

Reply:
Fixed.